docs/source/usage.rst

```diff
@@ -0,0 +1,50 @@
+.. _usage:
+
+Basic usage
+===========
+
+If you are a Geoscience ANALYST user, you can also skip this section since the import/export dialogs will be available
+through the menu system in ANALYST.  I you are not an ANALYST user, you should give it a try!  las-geoh5 will be
+included in the free viewer.  If you're *still* not convinced, or if you're an advanced user or developer and need to
+run a custom ui.json, then read on to learn how to run via ui.json.
```

Comment:
;) You want full-blown marketing department here. Would be tempted to leave it out, but we can let a third party decide.

Reply:
ABC dom.  Always-be-closing!

Reply:
removed